docs/csharp/whats-new/csharp-7-1.md

```diff
@@ -0,0 +1,176 @@
+---
+title: What's new in C# 7.1
+description: An overview of new features in C# 7.1.
+keywords: C# language design, 7.1, Visual Studio 2017, 
+author: billwagner
+ms.author: wiwagn
+ms.date: 08/16/2017
+ms.topic: article
+ms.prod: .net
+ms.devlang: devlang-csharp
+---
+
+# What's new in C# 7.1
+
+C# 7.1 is the first point release to the C# Language. This release is the
+first step toward the next major release of the langauge. This increased
+cadence enables you to use the new features sooner, ideally when
+each features is ready. This release also adds the ability to configure
+the compiler to match a specified version of the language. This feature
+separates the decision to upgrade tools from the decision to upgrade language
+versions.
+
+This release adds the [language version selectionn](#language-version-selection)
+configuration element, and four new language features.
+
+The new language features in this release are:
+
+* [`async` `Main` method](#async-main)
+  - The entry point for a console application can have the `async` modifier.
+* [`default` value expressions](#default-value-expressions)
+  - The `default(T)` expression may omit the target type when that type can be inferred.
+* [Inferred tuple element names](#inferred-tuple-element-names)
+  - The names of tuple elements can be inferred from tuple initialization in many cases.
+* [Reference assembly generation](#reference-assembly-generation)
+  - The `/refout` and `/refonly` compiler options enable reference assembly generation.
+
+## Langauge version selection
+
+The C# compiler that ships with Visual Studio 2017 V15.3 or with
+the .NET Core SDK 2.0 supports C# 7.1. However, the 7.1 features are turned
+off by default. To enable 7.1 features, you need to change the language
+version setting for your project.
+
+In Visual Studio, right-click on the project node in Solution explorer and select
+'Properties'. Select the "Build" tab and click on the "Advanced" button. In the dropdown,
+select "C# latest minor version (latest)", or the specific version "C# 7.1"
+as shwon in the image below. The 'latest' value means you want to use the latest
+minor versio on the current machine. The 'C# 7.1' means that you want to use C# 7.1,
+even after newer minor versions are released.
+
+[Setting the language version](./csharp-7-1/media/advanced-build-settings.png)
+
+Alternatively, you can edit the "csproj" file and add or modify the
+following lines:
+
+```xml
+<PropertyGroup>
```

Comment:
I think it might be useful to include a full <PropertyGroup> element including the Configuration attribute. That helps to make it clear that this is going to be a per-configuration setting.